formats/case_study/frontend/examples/archived.json

```diff
@@ -15,9 +16,15 @@
   "details": {
     "change_note": null,
     "tags": {
-      "browse_pages": [],
-      "topics": [],
-      "policies": []
+      "browse_pages": [
```

Comment:
The reason that's in this PR is that it's not possible to do these kind of mass-changes without reformatting first. I can't seem to remove the commit because subsequent commits depend on this one (can't find where). I'll try adding this as a separate PR so I can rebase on top of that.